easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2-CrayGNU-2016.03-cuda-7.0.eb

```diff
@@ -0,0 +1,44 @@
+# Contributed by 
+# Luca Marsella (CSCS)
+# Guilherme Peretti-Pezzi (CSCS)
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+
+name = 'GROMACS'
+version = "5.1.2"
+cudaversion = '7.0'
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.03'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = ' -DCMAKE_C_FLAGS="-g -fopenmp -fPIC" -DCMAKE_CXX_FLAGS="-g -fopenmp -fPIC" '
+configopts += ' -DCUDA_NVCC_FLAGS="-arch sm_35" '
+
+installopts = " -j16"
+ 
+skipsteps = ['test']
```

Comment:
We used to skip tests as they call the MPI executable gmx_mpi and they would fail systematically on the login nodes, where we usually build: we would need to submit a SLURM job to run the MPI executable on the compute nodes.